lib/graph/make_graph.js

```diff
@@ -90,8 +92,27 @@ module.exports = function(config, options){
 	});
 
 	return buildPromise.then(function(){
-		// When the build has loaded, update its config with any `buildConfig`
-		// options found.
+
+		var normalizedMetaPromises = _.map(configMetas, function(module){
```

Comment:
https://github.com/stealjs/steal-tools/pull/386/files#diff-e5bdcc156e0a2da2eb77e51a09c7b5ddR1687
in this test i have a package.json, that is loaded, i also add a config "meta"
but steal-tools load and normalize only the package.json, not the meta that pass though multibuild config.
if you remove my lines of code and run the test it("merge meta data with package.json", function(done){ jqueryt-module would not have bar=foo in the metadata